paths/brownian/CHANGELOG.md

```
@@ -0,0 +1,606 @@
+
+# [4.0.0-alpha.8](https://github.com/tsparticles/tsparticles/compare/v4.0.0-alpha.7...v4.0.0-alpha.8) (2026-01-22)
+
+**Note:** Version bump only for package @tsparticles/path-brownian
+
+# [4.0.0-alpha.7](https://github.com/tsparticles/tsparticles/compare/v4.0.0-alpha.6...v4.0.0-alpha.7) (2026-01-22)
+
+**Note:** Version bump only for package @tsparticles/path-brownian
+
+# [4.0.0-alpha.6](https://github.com/tsparticles/tsparticles/compare/v4.0.0-alpha.5...v4.0.0-alpha.6) (2026-01-22)
+
+### Bug Fixes
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)